package.json

```diff
@@ -21,8 +21,14 @@
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
-    ]
+      "react-app/jest",
+      "eslint:recommended",
+      "plugin:react/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "prettier/prettier": ["error", {"endOfLine": "auto"}]
+    }
   },
   "browserslist": {
```

Comment:
tool that allows specifying which browsers should be supported in your frontend app by specifying "queries" in a config file